meta/Meta.cpp

```diff
@@ -3817,6 +3817,7 @@ sai_status_t Meta::meta_generic_validation_create(
         // this is conditional attribute, check if it's required
 
         bool any = false;
+        bool all = true;
```

Comment:
maybe it's time to use this https://github.com/opencomputeproject/SAI/blob/master/meta/saimetadatautils.h#L194